ckanext/s3filestore/tests/test_plugin.py

```diff
@@ -62,13 +62,18 @@ def test_enqueueing_visibility_update(self):
                     kwargs={'visibility_level': 'private', 'pkg_id': 'abcde'}
                 )
             else:
-                from ckan.lib.jobs import DEFAULT_JOB_TIMEOUT
+                if toolkit.check_ckan_version('2.10'):
+                    timeout = toolkit.config.normalized('ckan.jobs.timeout')
```

Comment:
nice.
Does 2.10 also bring in the upgrade queue system with 'deadletter' endpoints and/or on_failure function injections yet?

Reply:
So this is upstream (inside ckan core) and we just need this for testing only?

Reply:
Pretty much. We want to capture the queueing call rather than let it through.